packages/discord-utilities/src/lib/limits.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * Namespace containing limits related to Discord channels.
+ */
+export const ChannelLimits = {
+	/**
+	 * Maximum characters allowed in a channel description.
+	 */
+	MaximumDescriptionLength: 1024,
+	/**
+	 * Maximum characters allowed in a channel name.
+	 */
+	MaximumNameLength: 100,
+	/**
+	 * Maximum viewers allowed per screen share.
+	 */
+	MaximumViewersPerScreenShare: 250
+};
```

Comment:
This here and everywhere else.
```suggestion
} as const;
```